ca/ca.go

```diff
@@ -164,54 +172,79 @@ type internalIssuer struct {
 	boulderSigner *bsigner.Signer
 }
 
-func makeInternalIssuers(issuers []bsigner.Config, lifespanOCSP time.Duration) (map[string]*internalIssuer, error) {
-	internalIssuers := make(map[string]*internalIssuer, len(issuers))
+func makeInternalIssuers(issuers []bsigner.Config, lifespanOCSP time.Duration) (issuerMaps, error) {
+	issuersByAlg := make(map[x509.PublicKeyAlgorithm]*internalIssuer, len(issuers))
```

Comment:
very much a nit but: shouldn't the length just be max 2? (one for RSA, one for ECDSA)

Reply:
Yep! Done.